### ISSUE TITLE
Model loss stuck at 0.0265

### ISSUE THREAD

**Issue by Coedu7** [asujaykk/Self-Driving-car#5]
So, im not sure what are the normal values for loss, but i have trained my models with two different batch sizes two times each and the loss doesn't seem to decrease further, the car won't turn properly and end up out of the track every time, i'm not sure if it is related to my pc specifications, but when i use the model provided here it runs perfectly on the tracks
![image](https://github.com/asujaykk/Self-Driving-car/assets/119018528/1286f65a-29cb-47c4-803e-8ac4cab86bbe)
Since i believe it may be because of my computer,
Intel I3 6100
8 RAM
No gpu

thinking on asking a friend to run the training on his pc, i know mine is pretty bad but it is what it is, i hope someone can guide help me

**Comment by asujaykk** [comment]
Could you please provide further details?
like learning rate, data sample size,  batch size
